MREUnityRuntime/MREUnityRuntimeLib/App/IMixedRealityExtensionApp.cs

```diff
@@ -133,5 +139,16 @@ public interface IMixedRealityExtensionApp
         /// </summary>
         /// <param name="actorId"></param>
         void OnActorDestroyed(Guid actorId);
+
+        /// <summary>
+        /// Send the event, using the current protocol
+        /// </summary>
+        /// <param name="update"></param>
+        void SendEvent(Payload update);
+
+        /// <summary>
+        /// Send the event, using the current protocol, with provide response ID
+        /// </summary>
+        void SendEvent(Payload update, string _responseId);
```

Comment:
Same as above. This is an internal detail to the runtime that shouldn't be touched by host app integrations.